viz-lib/src/visualizations/chart/Editor/ChartTypeSelect.jsx

```diff
@@ -20,6 +20,10 @@ export default function ChartTypeSelect(props) {
       result.push({ type: "custom", name: "Custom", icon: "code" });
     }
 
+    if (filterTypes) {
+      return result.filter(({ type }) => !filterTypes.includes(type));
```

Comment:
Few thoughts re. this:
1. this prop need a better name (not clear how it "filter" work: removes items or keeps them). Maybe sth like "hiddenTypes", "unavailableTypes" or whatever else you prefer;
2. I'd use some lodash here, like: `return filter(result, ({ type }) => !includes(filterTypes, type))`, for me it feels a bit safer, but it's up to you;
3. now this component needs some proptypes 😁